docs/team/alekkwek18.md

```diff
@@ -5,22 +5,39 @@ title: Alek Kwek's Project Portfolio Page
 
 ### Project: TalentConnect
 
-TalentConnect is a desktop address book application for freelancer third-party
-recruiters, with functionality to enhance the recruitment management experience.
-This includes matching job listings to candidates and tagging successful candidates
-to companies for billing.
+TalentConnect is a desktop address book application optimized for solo freelancer recruiters. It is a brownfield project adapted from [AddressBook Level 3](https://se-education.org/addressbook-level3/). The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: NIL
+* **New Feature**: Added `DeleteContactCommand`.
```

Comment:
Similar to `AddContactCommand`, not exactly a new feature, perhaps you could move it to Feature enhancement.